app/templates/agenda/agendaitems.hbs

```diff
@@ -33,6 +37,7 @@
       <Agenda::AgendaitemSearch
         @searchText={{readonly this.filter}}
         @onSearch={{this.searchAgendaitems}}
+        @fullwidth={{true}}
```

Comment:
`fullwidth` doesn't clearly reflect the intention of the argument for me. Maybe name it in line with the argument of `AuButton`?  `hideButtonText` or something like that?